zipline/pipeline/loaders/blaze/estimates.py

```diff
@@ -82,7 +82,10 @@ def __init__(self,
                  odo_kwargs=None,
                  data_query_time=None,
                  data_query_tz=None,
-                 checkpoints=None):
+                 checkpoints=None,
+                 split_adjustments_loader=None,
```

Comment:
Definitely agreed on `reader` instead of `loader`. As for generalizing from splits to adjustments: previously, we had only talked about applying split adjustments since our plan is to use this loader for earnings estimates, and EPS columns would be affected by splits but probably not by dividends. I guess it depends on whether or not we think we might use this loader for dividend-adjusted data. @jmccorriston